SUMMARY.md

```diff
@@ -23,6 +23,7 @@
 * [Node Cordon](reference/node-cordon.md)
 * [Node Drain](reference/node-drain.md)
 
+
```

Comment:
Is this empty line needed?

Reply:
not required, we can remove this @AJDatacore